Copter: update EDU650 reference frame (#5756)
ArduPilot/ardupilot_wiki: copter/source/docs/reference-frames-hexsoon-td650.rst

- Landing Gear
- Power Distribution Board

Additional Parts
----------------

- Compatible with the :ref:`Cube autopilot <common-thecube-overview>` with direct mounting screws for the standard carrier board
- 8000mAh to 12000mAh 6S battery
- RC transmitter with at least 6 channels

> - (Optional) four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to ease assembly

Reviewer: Could post the STL into the Wiki.  It's not a big file.

Author: Hi @peterbarker, yes, I thought of this but didn't know where to put it.

Reviewer: > Hi @peterbarker, yes, I thought of this but didn't know where to put it.

Probably right next to where you put the link in?  @Hwurzburg might have some ideas.

Author: ok, I guess you're suggesting we could actually upload the file to the wiki repo.. I've never done that.  If @Hwurzburg thinks that's OK I can do that... maybe as a follow-up PR?

Author: @Hwurzburg, ok.  I'll leave @peterbarker to do that upload if he's so inclined 'cuz he's the author.

Maybe we can just merge this as-is?  We've got the clamp link there already so it shouldn't be a blocker.

- (Optional) `solder sleeves <https://www.amazon.com/TICONN-Connectors-Waterproof-Insulated-Electrical/dp/B07GDDKJ1D>`__ to shorten ESC wires
- (Optional) 2cm x 50cm velcro battery straps to replace very short straps provided

Connection and Setup
--------------------

.. image:: ../images/reference-frames-hexsoon-edu650-assembly.jpg
    :target: ../_images/reference-frames-hexsoon-edu650-assembly.jpg

Connect the four ESC wires to the back of the autopilot as shown in the :ref:`QuadX configuration <connect-escs-and-motors>`

Parameter file: `hexsoon-td650.param <https://github.com/ArduPilot/ardupilot/blob/master/Tools/Frame_params/hexsoon-td650.param>`__

This parameter file can also be loaded using the Mission Planner's Config/Tuning >> Full Parameter Tree page by selecting "hexsoon-td650" from the drop down on the middle right and then push the "Load Presaved" button.

LED connection and configuration
--------------------------------

> The LED cable from each arm should be connected to any of PDB's six LED ports shown in green below (the order does not matter).

Reviewer: ```suggestion
The LED cable from each arm should be connected to any of PDB's six LED ports shown in green below (the order does not matter, but the LED ports must be filled lowest-to-highest).
```

Author: Txs, I've changed the wording to, "The LED cable from each arm should be connected to the PDB's LED ports labelled "1" to "4" and shown in green below (the order does not matter)."

The adjacent 6-pin port (highlighted in purple below) should be connected to two of autopilot's PWM output pins.

  .. image:: ../images/reference-frames-hexsoon-edu650-assembly-LEDs.png
      :target: ../_images/reference-frames-hexsoon-edu650-assembly-LEDs.png
      :width: 450px

> Then set the following parameters on the autopilot

Reviewer: ```suggestion
Then set the following parameters on the autopilot, assuming you have connected to AUX outputs 5 and 6.
```

Author: thanks, added something similar.

- :ref:`SERVO13_FUNCTION <SERVO13_FUNCTION>` = 129 (ProfiLED1)
- :ref:`SERVO14_FUNCTION <SERVO14_FUNCTION>` = 132 (ProfiLEDClock)
- :ref:`NTF_LED_LEN <NTF_LED_LEN>` = 8
- Check the :ref:`NTF_LED_TYPES <NTF_LED_TYPES>` bit for "ProfiLED"

  .. image:: ../images/reference-frames-hexsoon-edu650-params1.png
      :target: ../_images/reference-frames-hexsoon-edu650-params1.png

Build issues and advice
-----------------------

Below is a list of issues and recommendations

- Assembly is difficult because both upper and lower plates are required to hold the legs in place.  Temporarily use four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to hold the lower plate and legs together.

> - The ESC wires are too long resulting in messy wiring.  These can be shortened using `solder sleeves <https://www.amazon.com/TICONN-Connectors-Waterproof-Insulated-Electrical/dp/B07GDDKJ1D>`__

Reviewer 2: Where should we put the ESC's? One diagram shows them in the arms, another place zip tied them to the bottom plate.

Author: Txs.  In my vehicle i've mounted the PDB to the underside of the top plate.  I've shortened the power wires to the ESC and attached them to the underside of the top plate as well.  In my build at least, the ESCs would not fit within the arms.

I think it would be add some images of suggested mounting positions for all parts but I think we should first merge this as-is and then iterate with future improvements.

> - If a gimbal is to be used, the recommendation to place the battery between the upper and lower plates is impossible.  Solutions include:

Reviewer: I didn't see that recommendation.  I can't think that would work on this frame as you need a chunky battery to make it work.

Author: Hi @peterbarker, I thought the "recommendation" was in a glossy picture included with the kit which showed a gimbal attached.. but maybe [that was actually for the EDU**4**50](https://docs.cubepilot.org/user-guides/cubepilot-ecosystem/cubepilot-ecosystem-edu450-reference-design).  I've replaced the accusative line with a simpler recommendation on how to mount the battery.

>     - Use two smaller batteries mounted on the top of the frame on either side of the autopilot.  Becareful that the two batteries have the same voltage if connected in parallel

Reviewer: ```suggestion
    - Use two smaller batteries mounted on the top of the frame on either side of the autopilot.  Be careful that the two batteries have the same voltage if connected in parallel
```

Author: fixed, thanks.
